README.md

```diff
@@ -6,8 +6,8 @@
 This repo contains a React Native plugin for Square [Reader SDK]. Reader SDK for
 React Native supports the following native Reader SDK versions:
 
-  * iOS: 1.3.3 and above
-  * Android: 1.3.3 and above
+  * iOS: 1.4.9 and above
+  * Android: 1.4.4 and above
```

Comment:
Wondering why we choose to use different version for iOS and Android?

Reply:
From the changelogs, these are the latest different versions available from here:
https://developer.squareup.com/docs/changelog/mobile
Otherwise, 1.4.3 is the latest version which is available for both iOS and Android.